libs/blocks/global-navigation/global-navigation.js

```diff
@@ -7,11 +7,15 @@ import {
 } from '../../utils/utils.js';
 import { analyticsGetLabel } from '../../martech/attributes.js';
 import { toFragment } from './utilities.js';
+import { replaceKey } from '../../features/placeholders.js';
 
-const CONFIG = { search: { icon: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" focusable="false"><path d="M14 2A8 8 0 0 0 7.4 14.5L2.4 19.4a1.5 1.5 0 0 0 2.1 2.1L9.5 16.6A8 8 0 1 0 14 2Zm0 14.1A6.1 6.1 0 1 1 20.1 10 6.1 6.1 0 0 1 14 16.1Z"></path></svg>' } };
+const CONFIG = {
+  icons: {
+    company: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.46 118.11"><defs><style>.cls-1{fill:#fa0f00;}</style></defs><polygon class="cls-1" points="84.13 0 133.46 0 133.46 118.11 84.13 0"/><polygon class="cls-1" points="49.37 0 0 0 0 118.11 49.37 0"/><polygon class="cls-1" points="66.75 43.53 98.18 118.11 77.58 118.11 68.18 94.36 45.18 94.36 66.75 43.53"/></svg>',
```

Comment:
Nothing urgent or critical, but I think this icon can be optimized further - the viewbox size and styles can be removed from here and added in CSS

Reply:
Yup, I did that for another icon that ended up not being used, but I can do it here as well. However, this is a fallback icon for the logo block. I'm not sure if we want to have a hardcoded image in our code anyway. Instead, it should probably be added from Sharepoint by authors.